<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление финансами</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pin_code_button_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_9">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_clear_last">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>⌫</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_clear_all">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="input_pin_code">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>230</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>ВВЕДИТЕ ПИН-КОД</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_button_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 22px;
background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 22px;
background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 22px;
background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pin_code_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 22px;
background-color: #FFFFFF</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="wrong_pin_code">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>115</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
